src/codegen/llvm.zig

```diff
@@ -123,14 +124,30 @@ pub fn targetTriple(allocator: Allocator, target: std.Target) ![:0]u8 {
         .wasi => "wasi",
         .emscripten => "emscripten",
         .uefi => "windows",
-
+        .macos => "macosx",
+        .ios => "ios",
+        .tvos => "tvos",
+        .watchos => "watchos",
         .opencl,
         .glsl450,
         .vulkan,
         .plan9,
         .other,
         => "unknown",
     };
+    try llvm_triple.appendSlice(llvm_os);
+
+    if (target.os.tag.isDarwin()) {
+        const min_version = target.os.version_range.semver.min;
+        const version_str = try std.fmt.allocPrint(allocator, "{d}.{d}.{d}", .{
+            min_version.major,
+            min_version.minor,
+            min_version.patch,
+        });
+        defer allocator.free(version_str);
+        try llvm_triple.appendSlice(version_str);
```

Comment:
```suggestion
        try llvm_triple.writer().print("{d}.{d}.{d}", .{
            min_version.major,
            min_version.minor,
            min_version.patch,
        });
```

Reply:
Hahaha, ofc! Thanks Andrew, dunno why I missed this one...